File: commands/infos.go
package commands

import (
	"github.com/bwmarrin/discordgo"
	"github.com/caillouxetbatons/discord-bot/utils"
	"github.com/lus/dgc"
)

var Infos = dgc.Command{
	Name:        "infos",
	Description: "Affiche les informations du bot",
	Usage:       "infos",
	Example:     "infos",
	IgnoreCase:  true,
	Handler: func(ctx *dgc.Ctx) {
		e := utils.DefaultEmbed()
		e.Title = "Infos"
		e.Description = "Judy a été développée par <@168436075058954240> en utilisant la librairie [DiscordGo](https://github.com/bwmarrin/discordgo)."
		e.Thumbnail = &discordgo.MessageEmbedThumbnail{
			URL: ctx.Session.State.User.AvatarURL("256"),
		}

		_ = ctx.RespondEmbed(e)
	},
}

File: commands/infos.js
const EmbedUtils = require('../utils/EmbedUtils');

module.exports = {
    init: (builder, _) => {
        return builder.setDescription('Affiche les informations du bot.');
    },
    handleInteraction: (client, interaction) => {
        const embed = EmbedUtils.defaultEmbed()
            .setThumbnail(client.user.avatarURL())
            .setTitle("Infos")
            .setDescription(`
                ${client.user.username} a été développée par <@168436075058954240> en utilisant la librairie [Discord.js](https://discord.js.org).\n
                Son code est open-source et disponible [ici](https://github.com/AlexPresso/judy-discord-bot).
            `)

        interaction.reply({embeds: [embed]});
    }
}
